data/seeds/013-recipients.js

```diff
@@ -5,7 +5,7 @@ const recipients = [
     recipient_middle_name: 'tree',
     recipient_last_name: 'ross',
     recipient_date_of_birth: '1970-10-12',
-    recipient_veteran_status: false,
+    recipient_veteran_status: true,
```

Comment:
Good catch realizing the seed needed to be updated too!

In the future could implement Faker to create a large amount of seed recipients if DS needs it.